sdks/python/apache_beam/runners/interactive/interactive_beam.py

```diff
@@ -220,6 +220,24 @@ def display_timezone(self, value):
     """
     self._display_timezone = value
 
+  @property
+  def specified_cache_dir(self):
```

Comment:
Maybe we should call this `cache_root` and describe it as the "Root path specified by the user to store interactive cache files"?

Also @rohdesamuel , what do you think?

Reply:
Cache root sounds good

Reply:
Renamed to `cache_root`. Included your comment in our design-doc.